inference-engine/src/vpu/graph_transformer/src/stages/gather.cpp

```diff
@@ -23,10 +23,11 @@ void FrontEnd::parseGather(const Model& model, const ie::CNNLayerPtr& _layer, co
 
     IE_ASSERT(layer->axis < input->desc().numDims());
 
-    auto perm = DimsOrder::fromNumDims(input->desc().numDims()).toPermutation();
-    auto axis = perm[input->desc().numDims() - 1 - layer->axis];
+    const auto perm = DimsOrder::fromNumDims(input->desc().numDims()).toPermutation();
+    const auto ieNormalizedAxis = layer->axis < 0 ? input->desc().numDims() + layer->axis : layer->axis;
```

Comment:
Which code did it before? device part?

Reply:
Looks that we didn't support negative values in the plugin.
When I tried to run tests with a negative axis, internal error in plugin occurred.
By the way, functional tests from `tests_deprecated` have no cases with a negative axis

Reply:
From spec I see negative values are possible. If plugin does not support it - it should be tracked over separate ticket and solved

Reply:
#-34519